packages/storybook/src/generators/init/init.ts

```diff
@@ -85,8 +85,23 @@ function checkDependenciesInstalled(host: Tree, schema: Schema) {
     ) {
       devDependencies['@storybook/react'] = storybookVersion;
     }
-  }
 
+    if (isReactUsingWebpack5()) {
+      if (
+        !packageJson.dependencies['@storybook/builder-webpack5'] &&
+        !packageJson.devDependencies['@storybook/builder-webpack5']
+      ) {
+        devDependencies['@storybook/builder-webpack5'] = storybookVersion;
```

Comment:
So when someone runs `nx g migrate-to-webpack5`, do they also get it setup for storybook? Or do they have to run `nx g storybook-configuration`/ `nx g @nrwl/storybook:init` after running `migrate-to-webpack5`?

Reply:
there's a `migrate-to-webpack5` generator? 

But no :). this PR basically adds the fact that you can generate new Storybook setups that use Webpack 5 if the workspace uses webpack 5 for the React apps. + older storybook setups will show you a warning how to upgrade.

The problem with the storybook migrations is always the messy potentially highly customized Storybook JS config files 🤷 .

But my idea is to maybe have an opt-in generator in an upcoming PR, that can be launched manually and which tries to migrate you for the setups "we understand" if that makes sense